nautobot_golden_config/nornir_plays/config_backup.py

```diff
@@ -130,9 +130,14 @@ def config_backup(job_result, log_level, qs, device_to_settings_map):
             )
             logger.debug("Completed configuration from devices.")
 
-    except Exception as error:
+    except ValueError as error:
```

Comment:
Can we move this to the job and to a finally?

Reply:
Will test it.